.idea/misc.xml

```diff
@@ -4,10 +4,10 @@
   <component name="FrameworkDetectionExcludesConfiguration">
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="jbr-17" project-jdk-type="JavaSDK">
```

Comment:
Here again, I suggest that IDE settings should one's own -- and not in the online repo.

Reply:
I'm not sure about this. Those folks, who are working in IDEA must receive this files to re-store environment